content/includes/footer.html

```diff
@@ -59,10 +59,8 @@
 
   <div class="va-notice--banner va-notice--banner--alt">
     <div class="usa-grid">
-      <a id="feedback-tool" class="feedback-widget" tabindex="0" href="http://veterans.uservoice.com" data-uv-trigger="smartvote">
-        <span class="icon"></span>
-        <span class="text">Provide feedback on Vets.gov</span>
-      </a>
+      <h4>A new way to give us feedback</h4>
+      <p>Check back soon to share your feedback on what you'd like to see on Vets.gov.</p>
```

Comment:
It looks like we need a curly apostrophe here.

Reply:
Good call!